pkgs/development/python-modules/aws-secretsmanager-caching/default.nix

```diff
@@ -49,6 +49,6 @@ buildPythonPackage rec {
       The AWS Secrets Manager Python caching client enables in-process caching of secrets for Python applications.
     '';
     license = licenses.asl20;
-    maintainers = [ ];
+    maintainers = with maintainers; [ tomaskala ];
```

Comment:
This change should belong to the commit adding the aws-secretsmanager-caching package